docs/data-sources/omim.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: OMIM
+---
+
+import JSON from './omim-json.md'
+
+## Overview
+
+Nirvana uses gene symbols as the gene identifiers internally. To generate the OMIM database, we first map the MIM numbers, which are the primary identifiers used by OMIM, to gene symbols supported by Nirvana. Please note that there can be multiple MIM numbers mapped to one gene symbol. Only MIM numbers successfully mapped to a Nirvana gene symbol are further processed. The OMIM API is used to fetch all the information associated with a gene MIM number, except the gene symbols.
+
+## Download OMIM data
+
+To get access to the OMIM data, please set the "OmimApiKey" environment variable with a valid key value first. Then use the downloadOMIM command of SAUtils to download the OMIM data.
+
+```shell
+dotnet /path/to/SAUtils.dll downloadOMIM --ref  /path/to/Homo_sapiens.GRCh38.Nirvana.dat --uga /path/to/UGA.tsv.gz --out /path/to/output/
+```
+
+This command downloads three files:
+
+### 1. MimToGeneSymbol.tsv
+
+This file provides the mapping between MIM numbers and gene symbols. The complete list of MIM numbers is obtained from the mim2gene.txt ([http://omim.org/static/omim/data/mim2gene.txt](http://omim.org/static/omim/data/mim2gene.txt)). An example of this file is given below.
+
+```
+# MIM Number    MIM Entry Type (see FAQ 1.3 at https://omim.org/help/faq)   Entrez Gene ID (NCBI)   Approved Gene Symbol (HGNC) Ensembl Gene ID (Ensembl)
+100050  predominantly phenotypes
+100070  phenotype   100329167
+100100  phenotype
+100200  predominantly phenotypes
+100300  phenotype
+100500  moved/removed
+100600  phenotype
+100640  gene    216 ALDH1A1 ENSG00000165092
+100650  gene/phenotype  217 ALDH2   ENSG00000111275
+100660  gene    218 ALDH3A1 ENSG00000108602
+100670  gene    219 ALDH1B1 ENSG00000137124
+100675  predominantly phenotypes
+100678  gene    39  ACAT2   ENSG00000120437
+```
+
+The information in the "Entrez Gene ID (NCBI)",  "Approved Gene Symbol (HGNC)" and "Ensembl Gene ID (Ensembl)" columns are used to find the proper gene symbol supported by Nirvana, which may or may not be the same as the gene symbol listed here.  Every MIM number that has a Nirvana supported gene symbol will be put into the mapping file.download
```

Comment:
is there a .download file or is this a typo?

Reply:
It is a typo I think.